lib/solr/request/runner.rb

```diff
@@ -52,7 +52,9 @@ def call
 
       def solr_urls
         @solr_urls ||= begin
-          urls = if Solr.cloud_enabled? || Solr.master_slave_enabled?
+          urls = if Solr.node_url_override
+            [File.join(Solr.node_url_override, collection_name.to_s)]
```

Comment:
We better use addressable here, seems to be logically more correct

Reply:
yeah but the issue is that Addressable::URI.join will remove the path from the base url case it exists. For example
```
[3] pry(main)> Addressable::URI.join('http://localhost:8983/solr', 'test-core').to_s
=> "http://localhost:8983/test-core"
```

Reply:
What about parse and then join?

Reply:
same thing
```
[5] pry(main)> Addressable::URI.parse('http://localhost:8983/solr').join('/en')
=> #<Addressable::URI:0x3fca81816bf0 URI:http://localhost:8983/en>
```
unless we parse and then manipulate the parsed path, but it seems too much work for a just a  string concatenation.

Reply:
weird, ok then